### DISCUSSION TITLE
Unable to use the Falconpy

### DISCUSSION THREAD

**Discussion by eAwlaqi** [CrowdStrike/falconpy#626] [Q&A] [answered]
Hi all, 
I successfully installed falconpy package, please see screenshot below, however when I try to run 
`from falconpy import Hosts` 

I get this error

ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-154b37371795> in <module>
----> 1 from falconpy import Hosts

ModuleNotFoundError: No module named 'falconpy'
![falconpy](https://user-images.githubusercontent.com/45147849/165664388-42f995cd-d120-417f-9a0b-f63b0ea92a94.png)

I also checked the path, its "C:\**\someuser\AppData\Local\Programs\Python\Python310" and pip is also installed.
I am using windows 10 laptop, python 10.3

Any suggestions?
Thanks

**Comment by eAwlaqi** [comment, marked as answer]
Hey all, I created virtual environment, installed falconpy in it.. it worked.

Thanks

**Reply by jshcodes** [reply]
You beat me to it!  :tada:

This points out that we should do a better job documenting installation to virtual environments.  I'll get a ticket created for this. :grinning:

**Reply by jshcodes** [reply]
This issue can be tracked here: https://github.com/CrowdStrike/falconpy/issues/630

**Comment by eAwlaqi** [comment]
Thanks!

It was great learning to me as well.

Regards,

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
